packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -745,13 +752,7 @@ class IrisGridUtils {
    * @param  sorts The table sorts
    * @returns The dehydrated sorts
    */
-  static dehydrateSort(
-    sorts: Sort[]
-  ): {
-    column: ModelIndex;
-    isAbs: boolean;
-    direction: SortDirection;
-  }[] {
+  static dehydrateSort(sorts: readonly Sort[]): readonly DehydratedSort[] {
```

Comment:
Might not need to return readonly. But I can see the argument for sealing a value once it's dehydrated.